### ISSUE TITLE
"Could not find matches for regex expression" in dotNet Core generated AssemblyInfo.cs

### ISSUE THREAD

**Issue by ph1f1** [kyleherzog/AssemblyInfoReaderTask#20]
When using dotNet Core projects, the AssemblyInfo.cs file gets generated automatically from the *.csproj file.
It generates the AssemblyInfo properties in a format, which is not recognized by the used regex. It looks like the namespace part "System.Reflection" of the AssemblyAttribute is the problem as without it, it works just fine.

AssemblyInfo.cs generated by dotNet Core build:
```
//------------------------------------------------------------------------------
// <auto-generated>
//     This code was generated by a tool.
//
//     Changes to this file may cause incorrect behavior and will be lost if
//     the code is regenerated.
// </auto-generated>
//------------------------------------------------------------------------------

using System;
using System.Reflection;

[assembly: System.Reflection.AssemblyCompanyAttribute("xxx")]
[assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
[assembly: System.Reflection.AssemblyCopyrightAttribute("Copyright 2021")]
[assembly: System.Reflection.AssemblyDescriptionAttribute("xxx")]
[assembly: System.Reflection.AssemblyFileVersionAttribute("1.0.1.0")]
[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.1")]
[assembly: System.Reflection.AssemblyProductAttribute("xxx")]
[assembly: System.Reflection.AssemblyTitleAttribute("xxx")]
[assembly: System.Reflection.AssemblyVersionAttribute("1.0.1.0")]

// Generated by the MSBuild WriteCodeFragment class.
```

Messages from the Log:
```
Using CSharp/VbNet regex expression (?m)^\s*[\[\<]\s*[Aa]ssembly:\s*(\w*)\(\s*@?"([^"]*)
Could not find matches for regex expression (?m)^\s*[\[\<]\s*[Aa]ssembly:\s*(\w*)\(\s*@?"([^"]*) in content.
```

I guess this could easily be fixed by updating the regex expression accordingly.

**Comment by kyleherzog** [comment]
Just released a version 3.0 to address this issue.  Could you give it a try to confirm the fix?

**Comment by ph1f1** [comment]
The fix works fine, thanks!
